src/Protocol/Protocol.php

```diff
@@ -212,15 +212,19 @@ public function getAckFrame(Frame $frame, $transactionId = null)
      *
      * @param \Stomp\Transport\Frame $frame
      * @param string $transactionId
+     * @param bool $requeue
```

Comment:
please describe what this parameter is used for

Reply:
the requeue header on NACK will either requeue the message (default), or discard the message if false. if DLX is enabled for the queue, requeue=false will send the message to the deadletter exchange. see https://www.rabbitmq.com/dlx.html

Reply:
Please add a small note to the parameter doc, something like "Parameter are not available on all brokers".